src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -399,6 +402,32 @@ export default class DualThumb extends React.Component<Props, State> {
     }
   }
 
+  @autobind
+  private handleTrackClick(event: React.MouseEvent) {
+    const clickXPosition = this.actualXPosition(event.clientX);
+    let xPositionLower = this.props.min;
+    let xPositionUpper = this.props.max;
+
+    if (this.thumbLower.current && this.thumbUpper.current) {
+      xPositionLower = this.actualXPosition(
+        this.thumbLower.current.getBoundingClientRect().left,
+      );
+      xPositionUpper = this.actualXPosition(
+        this.thumbUpper.current.getBoundingClientRect().left,
+      );
+    }
+
+    if (clickXPosition > xPositionLower && clickXPosition < xPositionUpper) {
+      return;
```

Comment:
We won’t need an early return for out-of-bounds X positions because the event is tied to the track wrapper, which constrains the value so it is in-bounds.